pennylane/io/qasm_interpreter.py

```diff
@@ -26,7 +26,9 @@
     "Y": ops.PauliY,
     "Z": ops.PauliZ,
     "S": ops.S,
+    "SDG": lambda *args, **kwargs: ops.adjoint(ops.S(*args, **kwargs)),
```

Comment:
```suggestion
    "SDG": qml.adjoint(ops.S),
```
This should work... Can you check?

Reply:
Yes this works fine